File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -5820,7 +5149,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Certified signature 
 
-The following code snippet illustrates how to sign a PDF document without showing the digital signature.  
+The following code snippet illustrates how to sign a PDF document without showing the digital signature using [Certificated](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_Certificated) property.  
```

Comment:
The following code snippet illustrates how to sign a PDF document without showing the digital signature using [Certificated](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_Certificated) property in [PdfSignature]() class.

Reply:
Changed the content